src/utils/sprint-utils.ts

```diff
@@ -1,138 +1,125 @@
-import type { Person } from "src/generated/client";
-import type { Allocations, Projects, Tasks } from "src/generated/homeLambdasClient";
+import config from "src/app/config";
+import type { ResourceAllocations, ResourceAllocationsPhase, ResourceAllocationsProject, ResourceAllocationsUser, User, WorkHours } from "src/generated/homeLambdasClient";
 
 /**
- * Retrieve total time entries for an allocation
+ * Get project name
  *
- * @param allocation allocation
- * @param allocations list of allocations
- * @param timeEntries list of total time entries associated with allocations
+ * @param project project with type ResourceAllocationsProject
+ * @param projects list of projects with type ResourceAllocations[]
+ * 
+ * @returns project name
  */
-export const getTotalTimeEntriesAllocations = (
-  allocation: Allocations,
-  allocations: Allocations[],
-  timeEntries: number[]
+export const getProjectName = (
+  project: ResourceAllocationsProject,
+  projects: ResourceAllocations[]
 ) => {
-  if (timeEntries.length) {
-    return timeEntries[allocations.indexOf(allocation)] || 0;
-  }
-  return 0;
-};
+  const foundProject = projects.find(
+    (p) => p.project?.severaProjectId === project.severaProjectId
```

Comment:
Again project.project or even p.project isn't a great idea, see previous comments about this prop typing issue